tests/test_core.py

```diff
@@ -90,9 +91,31 @@ def test_stream_logs(self, openshift):
 
         (flexmock(openshift)
             .should_receive('_get')
-             # First: timeout in response after 100s
+            # First: timeout in response after 100s
+            .and_raise(ChunkedEncodingError(''))
+            # Next: return a real response
+            .and_return(response))
+
+        (flexmock(time)
+            .should_receive('time')
+            .and_return(0)
+            .and_return(100))
+
+        logs = openshift.stream_logs(TEST_BUILD)
+        assert len([log for log in logs]) == 1
+
+    def test_stream_logs_no_data(self, openshift):  # noqa
```

Comment:
Ah, correct, I was thinking 'no data after a timeout', but it would be too long. Fixed by swapping test names